File: lab1/src/main/java/Domain/BaseEntity.java
package Domain;

import org.jooq.Field;
import org.jooq.impl.DSL;

public class BaseEntity {
  private int id;
  public static Field<Integer> ID = DSL.field("id", Integer.class);

  public BaseEntity(int id) {
    this.id = id;
  }

  public BaseEntity() {
  }

  public int getId() {
    return id;
  }

  public void setId(int id) {
    this.id = id;
  }
}

File: proiect_c#/Domain/BaseEntity.cs
using System;

namespace csharp.Domain
{
	[Serializable]
	public class BaseEntity
	{
		public int Id { get; set; }

		public BaseEntity(int id)
		{
			this.Id = id;
		}

		public BaseEntity()
		{
		}
	}
}
